### PR TITLE
Added roman-numerals exercise.

### PR DESCRIPTION
**Pull request by cpaulbond**
Let me know if there are any problems!

### CODE REVIEW

**Comment by kytrinyx** [comment]
@canweriotnow do you have a moment to review this one?

**Comment by wobh** [comment]
The example implementation converts 5000 to "MMMMM" which should be invalid. Otherwise it seems to check out.

**Comment by kytrinyx** [comment]
Do we need to add a case to https://github.com/exercism/x-common/blob/master/roman-numerals.json?

File: exercises/roman-numerals/example.el [hunk 1]
```diff
@@ -0,0 +1,36 @@
+;;; roman-numerals.el --- roman-numerals Exercise (exercism)
+
+;;; Commentary:
+
```
Comment (yurrriq) [inline exercises/roman-numerals/example.el]: Personally, I prefer `when` over single branch `if`.

**Comment by canweriotnow** [comment]
I think it's good for now, capping the use case at decoding at 3k works, tests pass. Merging.

File: exercises/roman-numerals/example.el [hunk 2]
```diff
@@ -20,6 +20,8 @@
                   ("IV" .   4)
                   ("I"  .   1)))
         (roman nil))
+    (if (> value 3000)
```
Comment (wobh) [inline exercises/roman-numerals/example.el]: Why 3000? You can make Roman numerals up through 4999.

**Pull request merged**